build/docker/intel-deviceplugin-operator-ubi.Dockerfile

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 Intel Corporation. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# GOLANG_BASE can be used to make the build reproducible by choosing an
+# image by its hash:
+# GOLANG_BASE=golang@sha256:9d64369fd3c633df71d7465d67d43f63bb31192193e671742fa1c26ebc3a6210
+#
+# This is used on release branches before tagging a stable version.
+# The main branch defaults to using the latest Golang base image.
+ARG GOLANG_BASE=golang:1.17-bullseye
+
+FROM ${GOLANG_BASE} as builder
+
+ARG DIR=/intel-device-plugins-for-kubernetes
+ARG GO111MODULE=on
+ARG BUILDFLAGS="-ldflags=-w -s"
+WORKDIR $DIR
+COPY . .
+
+RUN cd cmd/operator; GO111MODULE=${GO111MODULE} CGO_ENABLED=0 go install "${BUILDFLAGS}"; cd -
+RUN install -D /go/bin/operator /install_root/usr/local/bin/intel_deviceplugin_operator \
+    && install -D ${DIR}/LICENSE /install_root/usr/local/share/package-licenses/intel-device-plugins-for-kubernetes/LICENSE \
+    && scripts/copy-modules-licenses.sh ./cmd/operator /install_root/usr/local/share/
+
+# FROM gcr.io/distroless/static
+FROM registry.access.redhat.com/ubi8-micro
+
+MAINTAINER NAME chaitanya.kulkarni@intel.com
```

Comment:
This was followed according to the Dockerfile requirements at [RH documentation](https://redhat-connect.gitbook.io/partner-guide-for-red-hat-openshift-and-container/program-on-boarding/technical-prerequisites) - Removed now since deprecated. Check on redhat portal and does not affect any test case